Sources/Kitura/RouterResponse.swift

```diff
@@ -116,6 +116,9 @@ public class RouterResponse {
         }
     }
 
+    /// User info.
```

Comment:
Although I understand this is a copy of the equivalent comment on `RouterRequest`, I think it would be worth fleshing out this comment, perhaps with some guidance on when someone might want to use it and/or a usage example.

In particular I think it would be useful to disambiguate 'user' in this context from the identity of a remote client.

Reply:
What about:

```swift
/// User info.
/// Can be used by middlewares and handlers to store and pass information on to subsequent handlers.
```

Reply:
Sounds good to me.  Would you mind adding this to the `RouterRequest` dictionary too, in this PR?